<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmuNWAccessTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txtIP">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblEmu">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,0,5">
      <item>
       <widget class="QGroupBox" name="grpCores">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Cores</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="lstCores"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="btnCoreLoad">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCoreUnload">
             <property name="text">
              <string>Unload</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btnCoreCurrentInfo">
             <property name="text">
              <string>Current Info</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCoreReset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btnCoreInfo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Info</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCoresList">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="grpEmu">
          <property name="title">
           <string>Emulation</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btnEmuInfo">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLoadGame">
             <property name="text">
              <string>Load Game ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEmuReset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEmuStop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEmuPause">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEmuResume">
             <property name="text">
              <string>Resume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEmuReload">
             <property name="text">
              <string>Reload</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpDebug">
          <property name="title">
           <string>Debug</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btnDebugBreak">
             <property name="text">
              <string>Break</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDebugContinue">
             <property name="text">
              <string>Continue</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpMemory">
        <property name="title">
         <string>Memory</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="cbxWriteMemory"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtWriteAddrLen">
             <property name="placeholderText">
              <string>addr;len;...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWrite">
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="txtWriteData">
           <property name="placeholderText">
            <string>Hex Data</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="cbxReadMemory"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtReadAddrLen">
             <property name="placeholderText">
              <string>addr;len;...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRead">
             <property name="text">
              <string>Read</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtReadData">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Reply</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="btnReadWriteTest">
             <property name="text">
              <string>Test</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="chkReadAppend">
             <property name="text">
              <string>Append</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
